src/common/utils/sandbox.ts

```diff
@@ -12,8 +18,24 @@
 export function showFn(contractId: string, abiFn: any) {
   return (
     abiFn.access !== 'private' &&
-    (abiFn.name !== 'stack-increase' ||
```

Comment:
It didn't make sense that we were making it invalid for function names to be "stack-increase". I adjusted the check to allow for the function name to be this, so long as the contract is not of the contracts listed

Reply:
I'd double check this, it seems to be related to a bug or problem that was reported a while ago.

Reply:
I reverted this